recipes/hopsmon.rb

```diff
@@ -0,0 +1,62 @@
+#Generate and sign certificates for hopsmon
+hopsworks_ip = private_recipe_ip('hopsworks', 'default')
+hopsworks_https_port = 8181
```

Comment:
Use 8182 the hopsworks/internal/port when you have to make requests within the cluster.